docs/plugins/analytics.md

```diff
@@ -52,11 +52,12 @@ learn how to contribute the integration yourself!
 The following table summarizes events that, depending on the plugins you have
 installed, may be captured.
 
-| Action     | Provided By    | Subject                                             |
-| ---------- | -------------- | --------------------------------------------------- |
-| `navigate` | Backstage Core | The URL of the page that was navigated to           |
-| `click`    | Backstage Core | The text of the link that was clicked on            |
-| `search`   | Backstage Core | The search term entered in any search bar component |
+| Action     | Provided By     | Subject                                             | Other Notes                                                       |
+| ---------- | --------------- | --------------------------------------------------- | ----------------------------------------------------------------- |
+| `navigate` | Backstage Core  | The URL of the page that was navigated to           |                                                                   |
+| `click`    | Backstage Core  | The text of the link that was clicked on            | The `to` attribute represents the URL clicked to                  |
+| `search`   | Search Plugin   | The search term entered in any search bar component | The `searchTypes` attribute holds `types` constraining the search |
+| `discover` | (By Convention) | The title of the search result that was clicked on  | The `value` is the result rank. A `to` attribute is also provided |
```

Comment:
I'm a bit unsure about the "provided by" column and more now when its mixed with core, plugins and something else. Is this meant to tell me what part of Backstage I can use this actions? If so can we clarify what "by convention" means here?

Reply:
Hmm...  Yeah, good point.  I guess the goal of the column is to inform _where_ the event is being fired from?  So that, if you saw the events start appearing in your analytics platform, you'd know where to look for what it meant.  ...Or if you are interested in being able to measure certain things, you'd know what you'd need to install to start getting the data.

...And from that perspective, `(By Convention)` is not super helpful.  And `Search Plugin` is kind of a misnomer too, because I think the event is actually fired within `@backstage/plugin-search-react`.  We could link directly to places in code where events are fired, but that'd be difficult to maintain.

Maybe we just get rid of the column entirely?  I'm not sure it's serving its purpose right now.